Sync-OneGetCode.ps1

```diff
@@ -0,0 +1,46 @@
+<# Sync dependencies from OneGet repo... #>
+param()
+$paths = @("Microsoft.PackageManagement\providers\inbox\Common\Extensions\Extensions.cs",
+           "Microsoft.PackageManagement\providers\inbox\Common\Utility\*.cs",
+           "Microsoft.PackageManagement\providers\inbox\Common\Version\*.cs",
+           "Microsoft.PackageManagement\Utility\Platform\OSInformation.cs")
```

Comment:
>icrosoft.PackageManagement\Utility\Platform\OSInformation.c [](start = 13, length = 59)

we possibly only need a small portion of OSInformation class in NuGet. If we can make it independent from OneGet, we do not this script.